deployments/ecs/README.md

```diff
@@ -1,3 +1,5 @@
+<!--- This content should be updated in the aws-ecs/README.md.jinja file. DO NOT MODIFY HERE. --->
```

Comment:
Is there going to be some kind of automatic sync process?  I'm not aware of any.  I don't think this comment is accurate.

Reply:
We request that any updates from Engineering be made in the Jinja file and not in this file. There will be no automatic sync process. I moved the information into the tile so that customers don’t have to go to GitHub for the setup information.